pinot-common/src/main/java/com/linkedin/pinot/common/restlet/resources/TableSegments.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.linkedin.pinot.common.restlet.resources;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+public class TableSegments {
+  List<String> segments;
+
+  public TableSegments(@JsonProperty("segments") List<String> segments) {
+    this.segments = segments;
+  }
+
+  public List<String> getSegments() {
+    return segments;
+  }
+
+  public void setSegments(List<String> segments) {
+    this.segments = segments;
+  }
+}
```

Comment:
We don't. There is not use for that right now and the classes are used only by jackson converter